routes/studentRoutes.js

```diff
@@ -4,7 +4,15 @@ var Student = require(path.join(__dirname, '../models/student'));
 
 routes.GETallStudents = function(req, res) {
   Student.find({}, function(err, allStudents) {
```

Comment:
you should just be able to do this filtering client side, rather than making multiple database requests